src/swarm/util/Verify.d

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
```

Comment:
Ahh, we'll have to keep removing this module whenever we merge :/ Not sure if there's a better way.

Reply:
I would just leave it there and remove it in v6.x.x.